### ISSUE TITLE
Bug when exporting ScatterPlotItem or PlotItem with Symbols using ImageExporter

### ISSUE THREAD

**Issue by mmeiler** [pyqtgraph/pyqtgraph#484]
Hi,

thanks for providing and maintaining this great visualization module.

When trying to export a plotItem that keeps a ScatterPlotItem or a PlotItem with symbols the symbols/ScatterPlotItems are not exported correctly.

The behavior can be reproduced when using the file [Plotting.py](https://github.com/pyqtgraph/pyqtgraph/blob/develop/examples/Plotting.py) in the examples directory.

If I use the export dialog the following images result:
**Entire Scene - no scaling**
![entirescene_noscale](https://cloud.githubusercontent.com/assets/28729656/26103962/2824be0e-3a3c-11e7-9af1-225855be5215.png)

**Entire Scene - scale width by factor 2 to 2000 px**
![entirescene_scale](https://cloud.githubusercontent.com/assets/28729656/26103969/2e40a410-3a3c-11e7-8c0e-bfa03a3f81f7.png)
_Please note that the symbols/markers are not visible any more._

**Plot 3 - no scaling**
![plot3_noscale](https://cloud.githubusercontent.com/assets/28729656/26103973/3302b9ac-3a3c-11e7-8602-444daeafbda0.png)
_Please note: markers/symbols not visible at all._

**Plot 3 - scale width to 1024 px**
![plot3_scale](https://cloud.githubusercontent.com/assets/28729656/26103978/3812b65e-3a3c-11e7-9b55-a91628c0487b.png)
_Please note: markers/symbols at a wrong location._

The same happens when using the API using 

```
exp = pyqtgraph.exporters.ImageExporter(p3)
exp.parameter()['width']=1024
exp.export('plot3_scale.png')
```

**Platform Information**

- sys.version: '2.7.3 (default, Mar 15 2017, 13:40:52) [MSC v.1600 64 bit (AMD64)]'
- platform.uname(): ('Windows', '', '7', '6.1.7601', 'AMD64', 'Intel64 Family 6 Model 94 Stepping 3, GenuineIntel')

Can you confirm whether this is a bug?

Thanks for your answer and best regards
Martin

**Comment by mmeiler** [comment]
Sorry, the pictures got lost:
here again for your reference
**Entire Scene - no scaling**
![entirescene_noscale](https://cloud.githubusercontent.com/assets/28729656/26103721/2104fb1c-3a3b-11e7-966c-eafbc9e46743.png)

**Entire Scene - scale width by factor 2 to 2000 px**
![entirescene_scale](https://cloud.githubusercontent.com/assets/28729656/26103730/25391d08-3a3b-11e7-9a48-c6af3e495001.png)

**Plot 3 - no scaling**
![plot3_noscale](https://cloud.githubusercontent.com/assets/28729656/26103735/29939b12-3a3b-11e7-8063-b860ebc8ba67.png)

**Plot 3 - scale width to 1024 px**
![plot3_scale](https://cloud.githubusercontent.com/assets/28729656/26103739/2ca26dc4-3a3b-11e7-8a73-79a4b0796cfa.png)

**Comment by paulmueller** [comment]
Looks like a duplicate of https://github.com/pyqtgraph/pyqtgraph/issues/105

**Comment by j9ac9k** [comment]
Fixed with #88 being merged
